settings/image.ini

```diff
@@ -76,6 +76,11 @@ AliasList[]=rss
 # to use image/jpeg for screenshot but rather image/png or image/gif.
 # MIMEType=image/png
 #
+# Override quality by MIME type for this alias only
+# This follows the pattern from the global [MIMETypeSettings] settings 
+# Quality[]
+# Quality[]=image/jpeg;80
```

Comment:
In PR description you mention `Quality[image/jpeg]=80` as format. This is the correct one as far as I can see?

Reply:
Sorry, I've fixed the PR description now! Good catch. I got distracted by the format of the internal map.